microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/advanced/AcceptHeadersAndFormatParameterInteractionTest.java

```diff
@@ -0,0 +1,365 @@
+import org.jboss.eap.qe.microprofile.openapi.v10.OpenApi10OnJaxRsAnnotationsTest;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.creaper.ManagementClientProvider;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+
+/**
+ * Test cases to verify negative scenario in which "Accept" HTTP header and {@code format} query parameter are
+ * conflicting or not valid.
+ * {@link MediaType} constants are used as HTTP "Accept" header values.
+ */
+@RunWith(Arquillian.class)
+@ServerSetup({ AcceptHeadersAndFormatParameterInteractionTest.OpenApiExtensionSetup.class })
+@RunAsClient
+public class AcceptHeadersAndFormatParameterInteractionTest {
+
+    private final static String DEPLOYMENT_NAME = OpenApi10OnJaxRsAnnotationsTest.class.getSimpleName();
+
+    static OnlineManagementClient onlineManagementClient;
+
+    @Deployment(testable = false)
+    public static Archive<?> createDeployment() {
+        WebArchive deployment = ShrinkWrap.create(
+                WebArchive.class,
+                String.format("%s.war", DEPLOYMENT_NAME))
+                .addClasses(ProviderApplication.class)
+                .addClasses(
+                        District.class,
+                        DistrictEntity.class,
+                        DistrictService.class,
+                        InMemoryDistrictService.class,
+                        DistrictsResource.class,
+                        RoutingServiceConstants.class)
+                .addAsWebInfResource(EmptyAsset.INSTANCE, "beans.xml");
+        return deployment;
+    }
+
+    static class OpenApiExtensionSetup implements ServerSetupTask {
+
+        @Override
+        public void setup(ManagementClient managementClient, String containerId) throws Exception {
+            //  MP OpenAPI up
+            onlineManagementClient = ManagementClientProvider.onlineStandalone();
+            OpenApiServerConfiguration.enableOpenApi(onlineManagementClient);
+        }
+
+        @Override
+        public void tearDown(ManagementClient managementClient, String containerId) throws Exception {
+            //  MP OpenAPI down
+            try {
+                OpenApiServerConfiguration.disableOpenApi(onlineManagementClient);
+            } finally {
+                onlineManagementClient.close();
+            }
+        }
+    }
+
+    /**
+     * @tpTestDetails Sends HTTP request with {@link MediaType#APPLICATION_JSON} {@code Accept} header and no {@code format}
+     *                query parameter.
+     * @tpPassCrit Despite missing {@code format} query parameter, the right Content Type is returned as per
+     *             MicroProfile OpenAPI spec.
+     *
```

Comment:
Could you remove this line in all JavaDocs, please?

Reply:
Ok, will do.